l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.py

```diff
@@ -194,11 +232,12 @@ def _get_datas(self):
             seq += 1
             sum_tax += line["vat_amount"]
             sum_turnover += line["turnover"]
-            vat = line["vat"].replace(" ", "").upper()
+            vat = line["vat"]
+            vat = self._format_vat_number(vat)
```

Comment:
I can't find where this method is defined.

Reply:
It disappeared in subsequent commits, I probably committed partial refactoring :/